packages/broker/src/plugins/operator/MaintainOperatorValueHelper.ts

```diff
@@ -0,0 +1,32 @@
+import { Logger } from '@streamr/utils'
+import { Contract } from 'ethers'
+import { Operator, StreamrConfig, operatorABI, streamrConfigABI } from '@streamr/network-contracts'
+import { OperatorServiceConfig } from './OperatorPlugin'
+
+const logger = new Logger(module)
+
+export class MaintainOperatorValueHelper {
+    config: OperatorServiceConfig
+    operator: Operator
```

Comment:
```suggestion
    private readonly config: OperatorServiceConfig
    private readonly operator: Operator
```
We can tighten the access modifiers here.